### ISSUE TITLE
-X still doesn't seem to work

### ISSUE THREAD

**Issue by somerandomgu** [Karunamon/muda#10]
Okay so this is what I am trying to do

" java -jar muda-2.0.0-DEV1.jar -t 8209 -b s -X 4 -N asd135 -d "C:\pathtothefiles" "

but when I hit enter it just post one file instead of 4. Also tried it with 5.

according to cmd I am using java version "1.8.0_71"

**Comment by Karunamon** [comment]
Hey there, thanks for the report.

I never got around to implementing -X unfortunately. 8chan's instability has made any kind of testing on this feature remarkably difficult. Someone else reported the same thing in #9.

**Comment by somerandomgu** [comment]
Damn, I really wish it would have worked. What about that muda-ruby thing? I couldn't get it to work, but if it worked, would it be able to post several pics in one post?

**Comment by Karunamon** [comment]
Generally that means that the site burped. That message comes up if the site returns a 404 for the thread. In a sane world, you'd only see that if the thread was really nuked, but in unstable-8chan-land, it comes up pretty often.

The ruby version *does* have working maxfile support, but it hasn't been tested in ages and I think it's somehow less tolerant of errors than this one. If you want to [download it and try](https://github.com/Karunamon/muda/tree/ruby), I can probably walk you through setting it up.

**Comment by somerandomgu** [comment]
Does the ruby version only work on linux? I can't seem to get it to work. Problem is I don't even know how to execute it.

./muda.rb -t 8830 -b s -d "C:\My Files\Pictures" does obviously not work, but I don't know what else to try either.

Also, isn't there a way to force it to continue? Like if there is a 404 error that it just ignores it and continues with the next file? Or sometimes it happens that it uploads a image, but it thinks it didn't upload it, then the script stops there because it can't continue. Is there a way to make it continue where it stopped the last time?

**Comment by Karunamon** [comment]
Yes on both counts. The first thing you need is a copy of ruby if you don't have one already - if you do, then the `./muda.rb` that you mentioned should Just Work assuming you did the bundle install stuff in that version's instructions.

As far as forcing it to continue, yes. Note that every post will have a number associated with it. If you run the exact same command again with `--resume (that number where it stopped)`, it should pick up where it left off. There's also `--maxfailures` that you could increase to a large number to work around it.

I can probably give you better instructions if you tell me exactly what error you're getting back from the script, though!

**Comment by somerandomgu** [comment]
I don't quite understand the -R command. What number do I have to write after it?

I put this into cmd 

java -jar muda-2.0.0-DEV1.jar -t 5648607 -b b -D 1 -x 500 -E 5 -d "G:\My Files\Pictures\2clovers.com\AquaBlanka-Part1-2clovers.com-2013"

script stops at 5/95 and says that the thread doesn't exist. Now if I want to continue what do I write? -R and then 6 because it stopeed at 5? Do I write the post id of the last post like 5670784?

**Comment by Karunamon** [comment]
Okay, so it stopped at image 5 - reuse the exact same command you used before, but add `-R 5` on it.

**Comment by somerandomgu** [comment]
When I write -resume it says unknown option. and when I try -R 5 it just starts at the first image like always. Does resume only work on ruby version?
